scipy/stats/_stats_mstats_common.py

```diff
@@ -3,10 +3,9 @@
 import numpy as np
 
 from . import distributions
-from . import futil
 
 
-__all__ = ['find_repeats', 'linregress', 'theilslopes']
+__all__ = ['_find_repeats', 'linregress', 'theilslopes']
```

Comment:
Yeah, OK. 
I've verified that even after `from scipy.stats import *` find_repeats is in the namespace, and `_find_repeats` is not.